services/channel/service/src/graphql/postgraphile-middleware.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  buildPgSettings,
+  LoginPgPool,
+  OwnerPgPool,
+} from '@axinom/mosaic-db-common';
+import {
+  AnnotateTypesWithPermissionsPlugin,
+  DeprecateStrayNodeIdFieldsPlugin,
+  enhanceGraphqlErrors,
+  forwardToGraphiQl,
+  getWebsocketFromRequest,
+  OmitFromQueryRootPlugin,
+  OperationsEnumGeneratorPluginFactory,
+  PostgraphileOptionsBuilder,
+  SubscriptionsPluginFactory,
+  ValidationDirectivesPlugin,
+} from '@axinom/mosaic-graphql-common';
+import {
+  AuthenticationConfig,
+  AxGuardPlugin,
+  EnforceStrictPermissionsPlugin,
+  getManagementAuthenticationContext,
+} from '@axinom/mosaic-id-guard';
+import {
+  customizeGraphQlErrorFields,
+  defaultWriteLogMapper,
+  getHttpServer,
+  getWebsocketMiddlewares,
+  Logger,
+  logGraphQlError,
+} from '@axinom/mosaic-service-common';
+import { StoreOutboxMessage } from '@axinom/mosaic-transactional-inbox-outbox';
+import PgSimplifyInflectorPlugin from '@graphile-contrib/pg-simplify-inflector';
+import { altairExpress } from 'altair-express-middleware';
```

Comment:
If we really need altair endpoint - please add `altair-express-middleware` as an
explicit dependency to the project. Also, I wonder if there is a tool out there
that can go through the project, check all used 3-rd party dependencies, and
check which of them are not defined in `package.json`... If there is - we might
want to introduce that to our PR build pipelines.

Reply:
`npx depcheck` - it helps to find missing ones and suggests ones that are not used. But it is not perfect e.g. we use the GraphQL codegene but only as part of our scripts so it is not possible for the tool to find it.